### DISCUSSION TITLE
Changing/appending to values of insert query

### DISCUSSION THREAD

**Discussion by ArihantRk** [JSQLParser/JSqlParser#1575] [Q&A] [answered]
Hi,

Can we parse the values the of insert query and append to any the of values

Example Query:
insert into employee values("test","data")

1. From above example, i want to append **data** value to **data_of_emplee**
2. Also, can we identify type of value like **test** is type is **string**

**Comment by wumpz** [comment, marked as answer]
Look into 
https://github.com/JSQLParser/JSqlParser/wiki/Examples-of-SQL-building#replacing-string-values 
this should give you an idea of how to achieve this.

However, you should know that JSqlParser treats double-quoted identifiers as object names and not string literals. Therefore you get for *"test"* and *"data"* a column object. Consider using single quotes.
